modules/gtk/gtk_mod.c

```diff
@@ -784,7 +784,7 @@ int gtk_mod_connect(struct gtk_mod *mod, const char *uri)
 
 	err = account_uri_complete(ua_account(mod->ua_cur), uribuf, uri);
 	if (err)
-		return EINVAL;
+		goto out;
```

Comment:
can you please make a separate PR for this ?

it makes it easier to read the main patch

also we should consider moving "gtk" to baresip-apps.